api/ivy.xml

```diff
@@ -22,6 +22,10 @@
 
 		<dependency org="org.springframework.security"   name="spring-security-core"      rev="${dependency.spring.security.revision}" transitive="false"/>
 		<dependency org="org.springframework"            name="spring-beans"              rev="${dependency.spring.framework.revision}"         transitive="false"/>
+		
+		<dependency org="org.slf4j" name="slf4j-api" rev="1.7.3" transitive="false"/>
+    	<dependency org="org.slf4j" name="slf4j-log4j12" rev="1.7.3" transitive="false"/>
```

Comment:
Ah you mean in a test? You shouldn't need a binding SLF4J just goes silent if no binding is present:
http://www.slf4j.org/manual.html